http/src/main/resources/reference.conf

```diff
@@ -603,13 +603,17 @@ gke {
     dbPassword = "replace-me"
     enabled = true
   }
-  rstudioApp {
+  aouApp {
```

Comment:
Would be good to keep the terminology consistent. The app type is `ALLOWED`, so maybe this should be `allowedApp { … }`?